src/lib-components/CardMeta.vue

```diff
@@ -107,10 +129,16 @@ const parsedLocations = computed(() => {
     }
   })
 })
+
+// if we want to use theme pattern based on passing classes prop to each component
+const parsedClasses = computed(() => {
+  // (maybe classes.join() if need an array)
+  return ['card-meta', theme?.value || '', props.isCentered ? 'centered' : '']
+})
 </script>
 
 <template>
-  <div class="card-meta">
+  <div :class="parsedClasses">
```

Comment:
Rename parsedClasses to classes, like other components toplevel computed class method